interfaces/xenmgr.xml

```diff
@@ -131,6 +131,12 @@
       <arg name="uuid" type="s"/>
       <arg name="obj_path" type="o"/>
     </signal>
+	
+	<signal name="notify">
+	  <tp:docstring>Send out generic notification, details inside string arg.</tp:docstring>
+      <arg name="uuid" type="s"/>
+      <arg name="messsage" type="s"/>
+	</signal>
```

Comment:
The formatting needs to be fixed up a bit. Looks like tabs mixed with spaces etc.